.github/workflows/ci.yaml

```diff
@@ -58,69 +63,39 @@ jobs:
           - tls-integration
           - client-integration
           - ha-integration
+        runner: ["ubuntu-22.04"]
+        include:
+          - tox-environments: h-scaling-integration
+            runner: "Ubuntu_4C_16G"
```

Comment:
Is `Ubuntu_4C_16G` used only for `h-scaling-integration` or for all the tests?

Reply:
This is the "big runner" - we only have one, and we're sharing it with other teams, so we're only using it to run the horizontal scaling tests, since they need to run on a more powerful machine.